README.md

```diff
@@ -7,62 +7,28 @@
 <p align="center"> 
 A handy data mocking api to generate data in  given schema
 </p>
+<div align="center">
 
-<p>
-  <a href="https://acmvit.in/" target="_blank">
-    <img alt="made-by-acm" src="https://img.shields.io/badge/MADE%20BY-ACM%20VIT-blue?style=for-the-badge" />
-  </a>
-    <img alt="license" src="https://img.shields.io/badge/License-MIT-green.svg?style=for-the-badge" />
-    <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/ACM-VIT/dolos?style=for-the-badge">
-    <img alt="GitHub repo size" src="https://img.shields.io/github/repo-size/ACM-VIT/dolos?style=for-the-badge"/>
-    <img alt="GitHub forks" src="https://img.shields.io/github/forks/ACM-VIT/dolos?style=for-the-badge"/>
-    <img alt="GitHub stars" src="https://img.shields.io/github/stars/ACM-VIT/dolos?style=for-the-badge"/>
-    <img alt="Github Actions" src="https://github.com/ACM-VIT/dolos/workflows/Node.js%20CI/badge.svg" />
-</p>
-<img alt="GitHub Hacktoberfest combined status" src="https://img.shields.io/github/hacktoberfest/2020/ACM-VIT/dolos?style=for-the-badge">
+[![made-by-acm](https://img.shields.io/badge/MADE%20BY-ACM%20VIT-blue?style=for-the-badge)](https://acmvit.in/)
+![GitHub last commit](https://img.shields.io/github/last-commit/ACM-VIT/dolos?style=for-the-badge)
+![GitHub repo size](https://img.shields.io/github/repo-size/ACM-VIT/dolos?style=for-the-badge)
+[![License](https://img.shields.io/badge/License-MIT-green.svg?style=for-the-badge)](https://en.wikipedia.org/wiki/MIT_License)
+[![GitHub forks](https://img.shields.io/github/forks/ACM-VIT/dolos?style=for-the-badge)](https://github.com/ACM-VIT/dolos/network)
+[![GitHub stars](https://img.shields.io/github/stars/ACM-VIT/dolos?style=for-the-badge)](https://github.com/ACM-VIT/dolos/stargazers)
+![Github Actions](https://github.com/ACM-VIT/dolos/workflows/Node.js%20CI/badge.svg)
```

Comment:
You can change the src for:

`https://img.shields.io/github/workflow/status/ACM-VIT/dolos/Node.js%20CI?style=for-the-badge`

and your PR closes #42 too

Reply:
c4df8ed closes #42